src/scripts/cfo.zig

```diff
@@ -242,7 +240,11 @@ fn run_fuzzers(
                 };
 
                 if (fuzzer_done or last_iteration) {
-                    log.debug("will reap '{s}'", .{fuzzer.seed.command});
+                    log.debug(
+                        "will reap '{s}'{s}",
+                        .{fuzzer.seed.command},
+                        if (!fuzzer_done) " (timeout)" else "",
```

Comment:
Ah last string, this needs to be in the arguments `.{}` list.

Reply:
one of this days, I'll either hook all "does it compiles/is it formatted" checks to `./zig/zig build check`, OR I make `./zig/zig build test` to do all that plus finish in seconds!

Right now, I routinely slip many trivial mistakes in PRs simply because its easier to `git push` then to think about which quick command I need to run to check if my code compiles :)